{{cookiecutter.project_slug}}/.circleci/config.yml

```diff
@@ -146,6 +146,16 @@ jobs:
           paths:
             - vendor/bundle
 
+      - run:
+          name: Install CocoaPods
+          command: |
+            curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
+            bundle exec pod install
+       - save_cache:
+          key: 1-pods-{{ checksum "Podfile.lock" }}
+          paths:
+            - ios/Pods
+
```

Comment:
This is just a copy and paste of the Circle CI change by Manu.

Reply:
This was already added by Manu. Merge in master and you'll get it.